packages/uui-radio/lib/uui-radio.element.ts

```diff
@@ -59,6 +59,7 @@ export class UUIRadioElement extends LitElement {
         border-radius: 100%;
         margin-right: calc(var(--uui-size-2) * 2);
         position: relative;
+        flex: 0 0 var(--uui-radio-button-size);|
```

Comment:
It seems a `|` character snuck in to the end of this line

Reply:
Good spot